Cargo.toml

```diff
@@ -1,36 +1,44 @@
 [package]
```

Comment:
Not that it's a problem, but I'm curious: did you use a tool to re-format `Cargo.toml`?

Reply:
I use the `cargo-edit` tool to add dependencies, which also automatically formats it.